packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -1137,21 +1135,21 @@ class _SearchFieldSuffix extends StatelessWidget {
 }
 
 class SearchNavigationControls extends StatelessWidget {
-  const SearchNavigationControls(this.controller, {@required this.onClose});
+  const SearchNavigationControls(this.controller, {required this.onClose});
 
-  final SearchControllerMixin controller;
+  final SearchControllerMixin? controller;
 
-  final VoidCallback onClose;
+  final VoidCallback? onClose;
 
   @override
   Widget build(BuildContext context) {
     return ValueListenableBuilder(
-        valueListenable: controller.searchMatches,
-        builder: (context, matches, _) {
+        valueListenable: controller!.searchMatches,
+        builder: (context, List<DataSearchStateMixin?> matches, _) {
```

Comment:
why does DataSearchStateMixin need to be nullable?

Reply:
done